kvclient_test.go

```diff
@@ -32,19 +32,25 @@ func TestKvClientReconfigureBucket(t *testing.T) {
 	}
 
 	cli, err := NewKvClient(context.Background(), &KvClientConfig{
-		Logger:        logger,
```

Comment:
Why? We should probably add not remove logging?

Reply:
We didn't actually remove it.  The Logger moved to KvClientOptions instead of KvClientConfig, and thus it's 'inherited' from the KvClientPool (as KvClientPool inherits from KvClientManager) rather than it being specified independently here.